packages/js/product-editor/src/hooks/use-variation-switcher.ts

```diff
@@ -66,6 +72,7 @@ export function useVariationSwitcher( {
 			parentProductType || 'product',
 			parentId,
 		] );
+		invalidateResolutionForStoreSelector( 'getProductVariations' );
```

Comment:
I think that we should also invalidate the `getProductVariationsTotalCount` resolution as well, to update the variation without price count.

![Screenshot 2023-10-20 at 11 32 36](https://github.com/woocommerce/woocommerce/assets/1314156/c65db468-94e9-4b5a-9a42-9e227126f8c1)

Reply:
done -> https://github.com/woocommerce/woocommerce/pull/40900/commits/1838e63fafab328620ab78b46518197953b262f3